hw3/01-characters.js

```diff
@@ -1,4 +1,71 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
+let app = document.querySelector('#results');
 
-fetch(url);
+app.style.display = 'flex';
+app.style.flexBasis = 'auto';
+app.style.flexDirection = 'row';
+app.style.flexFlow = 'row wrap';
+app.style.alignItems = 'column';
+app.style.width = '100%';
+app.style.margin = '0 auto';
+
+const addCharacterToDom = (character) => {
+  let element = document.createElement('figure');
+  let Picture = document.createElement('img');
+  let name = document.createElement('figcaption');
+  let title = document.createElement('figcaption');
+
+  element.style.display = 'flex';
+  element.style.flexDirection = 'column';
+  element.style.alignItems = 'center';
+  element.style.width = '25%';
+  element.style.margin = '0 auto';
```

Comment:
I think it would be a good idea to add a little bit of padding to the figure, especially on top.